### ISSUE TITLE
[BUG] Beanie persists `root` field

### ISSUE THREAD

**Issue by WingCode** [roman-right/beanie#668]
**Describe the bug**
Beanie persists root field in the database when using Pydantic RootModel. The root field must not be persisted in the database.

**To Reproduce**
```python
from typing import List
from motor.motor_asyncio import AsyncIOMotorClient
from pydantic import RootModel
from beanie import Document, init_beanie

Pets = RootModel[List[str]]

class PetDocument(Document):
    content: Pets

async def example():
    client = AsyncIOMotorClient("mongodb://user:pass@host:27017")

    await init_beanie(database=client.db_name, document_models=[PetDocument])

    my_pets = PetDocument(content=["dog", "cat", "fish"])

    await my_pets.insert()

import asyncio
asyncio.run(example())
```

**Expected behavior**
In MongoDB it is expected to have record like
```
{"content": ["dog", "cat", "fish"]
```
but instead it stores it as
```
{"content": {"root":["dog", "cat", "fish"]}}

**Comment by roman-right** [comment]
Fixed in https://github.com/roman-right/beanie/pull/669 . Please, try

**Comment by WingCode** [comment]
This has been fixed in #669 . Thank you for looking into it!
